### ISSUE TITLE
`brew` install for M2 release

### ISSUE THREAD

**Issue by pchiusano** [unisonweb/unison#1989]
This depends on #2020

**Comment by pchiusano** [comment]
Want to create some sort of launch script to pass the appropriate path to the ui and install that launch script in `/bin` or whatever.

**Comment by aryairani** [comment]
This is the Azure script we currently use to generate and update the brew formula with each M1 release. https://github.com/unisonweb/unison/blob/series/M1/azure-pipelines.yml

Here is the repo we keep the formula in:
https://github.com/unisonweb/homebrew-unison

**Comment by aryairani** [comment]
I'm happy to pair on this if you want.

**Comment by pchiusano** [comment]
Yeah I might need a hand. I think we want to split apart this workflow - #2020 is responsible for building the release, and then (somehow?) after that's done, this just runs and uploads the new brew formula. 

I feel like someone must have done this before (build a homebrew formula automatically from a github release) but I didn't manage to find anything via google. @hojberg did you have some ideas about where to find such things?

**Comment by aryairani** [comment]
@pchiusano The Azure script linked above does that, and the Github Actions version is likely very similar given the Microsoft connection; if that’s what you mean.

**Comment by pchiusano** [comment]
Also linking to this to look into: https://github.com/marketplace/actions/homebrew-tap

**Comment by aryairani** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew
https://docs.brew.sh/Taps
https://docs.brew.sh/Bottles
https://docs.brew.sh/Formula-Cookbook
https://docs.brew.sh/Tips-N'-Tricks
https://docs.brew.sh/Acceptable-Formulae
https://docs.brew.sh/Manpage
https://rubydoc.brew.sh/Formula.html
https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html
https://stackoverflow.com/a/51361039/310162

**Comment by aryairani** [comment]
Closed by unisonweb/homebrew-unison@bb9316e01281041fa7a200d2f9f4d62fb6ed0cab

**Comment by pchiusano** [comment]
@aryairani will we remove or replace the non `--head` version with M2 once that is out? If so do you want to track that in a separate ticket?

**Comment by aryairani** [comment]
Yes, once we have the final build.  I reopened the ticket.

**Comment by aryairani** [comment]
Closed by unisonweb/homebrew-unison@63b8c41

```
arya@jrrr unisonweb % brew install unison-language       
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).
==> Updated Formulae
Updated 15 formulae.
==> Updated Casks
Updated 5 casks.

==> Installing unison-language from unisonweb/unison
==> Downloading https://github.com/unisonweb/unison/releases/download/M2-macOS/unison-macOS.tar.gz
Already downloaded: /Users/arya/Library/Caches/Homebrew/downloads/fb276b019a1a082e0951a7b08d325da7f032826b2a7665ec69c0e3e482407b11--unison-macOS.tar.gz
🍺  /opt/homebrew/Cellar/unison-language/1.0.M2: 11 files, 48.6MB, built in 2 seconds
Removing: /opt/homebrew/Cellar/unison-language/1.0.M1m... (4 files, 22.5MB)
Removing: /Users/arya/Library/Caches/Homebrew/unison-language--HEAD.tar.gz... (11.4MB)
arya@jrrr unisonweb % ucm -version
ucm version: release/M2
arya@jrrr unisonweb %
```
